### ISSUE TITLE
Why is the reproduction accuracy not as good as in the paper?

### ISSUE THREAD

**Issue by ElonZhou99** [yjh0410/PyTorch_YOWO#3]
Why is the reproduction accuracy not as good as in the paper?

**Comment by yjh0410** [comment]
@ElonZhou99 Not as good as in the paper? From the results reported in this project, my YOWO is better than the official YOWO. I am a little confused about your question. Can you tell me more about it?

**Comment by ElonZhou99** [comment]
> @ElonZhou99 Not as good as in the paper? From the results reported in this project, my YOWO is better than the official YOWO. I am a little confused about your question. Can you tell me more about it?

I'm apologize that i got a mistake.

**Comment by ElonZhou99** [comment]
@yjh041 The new question: Do the train epoch setting(epoch = 4) enough? In my training, the first epoch got mAP66%.

**Comment by yjh0410** [comment]
@ElonZhou99 That's OK.

On the UCF101-24, I train YOWO and YOWO-Nano with only 5 epochs, so I think epoch=4 is enough.

**Comment by yjh0410** [comment]
@ElonZhou99  Hi ! Dear friend, I train my YOWO on UCF101-24 with 1 epoch, and then I evaluate it on UCF101-24. The mAP results are follows.

```Shell
AP: 75.67% (1)
AP: 97.96% (10)
AP: 81.94% (11)
AP: 63.31% (12)
AP: 71.84% (13)
AP: 94.22% (14)
AP: 87.91% (15)
AP: 82.89% (16)
AP: 81.01% (17)
AP: 84.03% (18)
AP: 94.29% (19)
AP: 45.15% (2)
AP: 87.88% (20)
AP: 70.56% (21)
AP: 80.81% (22)
AP: 68.45% (23)
AP: 86.82% (24)
AP: 84.21% (3)
AP: 70.23% (4)
AP: 46.04% (5)
AP: 95.36% (6)
AP: 92.70% (7)
AP: 87.73% (8)
AP: 92.77% (9)
mAP: 80.16%
```

As you can see, the mAP is 80.16%, so I am a little confused about your 66% mAP. I wonder whether you might have any trouble training YOWO.

**Comment by ElonZhou99** [comment]
@yjh0410 wow，just 1 epoch can get so high mAP!  In my test, I changed the 2D backbone and I did not load any pre-train weight for 2D and 3D backbone. Maybe this is the reason that i got a lower mAP.

**Comment by ElonZhou99** [comment]
@yjh0410 And could you test the epoch 1 mAP without any pre-train weight of the backbone? 
Thank you!

**Comment by yjh0410** [comment]
@ElonZhou99 I agree with you. No pretrained weight of 2D or 3D backbones might be the major reason. I recommend you load the pretrained weight of the 3D backbone since the 3D CNN is hard to be trained well.

**Comment by ElonZhou99** [comment]
> @ElonZhou99 I agree with you. No pretrained weight of 2D or 3D backbones might be the major reason. I recommend you load the pretrained weight of the 3D backbone since the 3D CNN is hard to be trained well.

Ok, thank you.
